packages/mux-player/src/index.ts

```diff
@@ -99,6 +99,7 @@ const playerSoftwareName = 'mux-player';
 
 class MuxPlayerElement extends VideoApiElement {
   #tokens = {};
+  #userInactive: boolean;
```

Comment:
we need to store it for now because Media Chrome can fire multiple events for user inactive changed, without the value actually changing right now, so, we want to work around that.